### PR TITLE
Contracts disclosure to publication

### PR DESCRIPTION
**Pull request by DRH-TBS**

### CODE REVIEW

File: ckanext/canada/tables/contracts.yaml [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
+    données qui sont soumis à des règles de validation des données conformément
+    au Guide de la publication proactive des marchés.
+    Les utilisateurs devraient examiner les règles de validation des données dans
+    l’onglet de référence du modèle pour chaque champ de données sous «
+    validation » pour plus d’informations afin de s’assurer que les données peuvent
+    être téléchargées dans le registre.
 
 template_version: 3
-template_updated: 2023-03-14
+template_updated: 2023-08-14
```
Comment (wardi) [inline ckanext/canada/tables/contracts.yaml:24]: August?

**Comment by wardi** [comment]
There are a number of other changes to fields coming as well, right?

File: ckanext/canada/tables/contracts.yaml (hunk 1 above)
Comment (DRH-TBS) [reply ckanext/canada/tables/contracts.yaml:24]: That is what the client indicated. Copied from the document.

**Comment by DRH-TBS** [comment]
> There are a number of other changes to fields coming as well, right?

Yes, more changes for another ticket due next week.

**Pull request merged**